cpp/src/holtwinters/holtwinters_utils.hpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
Let's just make a single `holtwinters_utils.h` file and put all the implementations in there. We shouldn't need to define everything in a separate header/impl since it's unlikely the implementations are only used internally to this algorithm.